apps/web/app/api/organization-team-employee/[id]/route.ts

```diff
@@ -15,15 +15,15 @@ export async function PUT(req: Request, { params }: { params: { id: string } })
 
 	const { id } = params;
 
+	const response = await updateOrganizationTeamEmployeeRequest({
```

Comment:
@paradoxe35 that code should be inside the `if` statement, no?

Reply:
@paradoxe35 not sure I got why we want that etc, let's discuss it later in Slack, for now will merge :)